.github/workflows/e2e-tests.yml

```diff
@@ -38,7 +38,7 @@ jobs:
       - name: Install dependencies and build
         run: |
           npm run ci:install
-          npm run ci:setup

```

Comment:
ci:setup Will have different behavior if running for dev, staging or prod. Not sure we want to put here directly the actual code instead of a kind of alias.

Reply:
How bout once we know for sure what the different behavior will look like between each env, at that point we change the script? Right now all it was doing was calling the build script

Reply:
ok